app_server/app/Router.js

```diff
@@ -97,6 +97,9 @@ server.register(require('inert'), function(err) {
   server.route({
     method: 'GET',
     path: '/admin/{param*}',
+    config: {
+      auth: false
+    },
```

Comment:
Ah okay, would be nice if only the login page is publicly accessible while other pages will give `Access Denied`. But you can do this in the future if i possible. Other than this, should be good to merge.